storage/store.go

```diff
@@ -477,14 +477,17 @@ func (s *Store) Start(stopper *util.Stopper) error {
 		s.ctx.Gossip.RegisterCallback(capacityRegex, s.capacityGossipUpdate)
 	}
 
+	// Update the store status values.
+	status := &proto.StoreStatus{
+		StoreID:   s.Ident.StoreID,
+		NodeID:    s.Ident.NodeID,
+		StartedAt: now.WallTime,
+	}
+	atomic.StorePointer(&s.status, unsafe.Pointer(status))
```

Comment:
I'm suggesting move this code to before line 467. Otherwise updating of the status races with the range scanner.